src/articles/FindArticles.js

```diff
@@ -34,6 +91,12 @@ export default function NewArticles({ api }) {
 
   return (
     <>
+      {!hasExtension && Feature.extension_experiment1() && !displayedExtensionPopup ? (
```

Comment:
May I ask one more thing? 

Look at the Reminder: it hides the complexity of the conditionals of when it shows itself inside of the component. 
We could do the same here, no? Code would be nicer here, and the behavior related to the extension message, including when it should be shown or not, would be located in that component.

Do you see what I mean?

Reply:
Final observation here: if you look at who uses the `isChrome` state var it's only the Reminder. 
It never changes it, so it's not exactly state either.

Thus, you can remove that state var completely it seems to me, and you can simply call the `runningInChromeDesktop` from within the `Reminder` methinks. 

```js
if (
    !hasExtension &&
    runningInChromeDesktop() &&
    Feature.extension_experiment1()
  ) {
```

See what I mean?